vike-vue/package.json

```diff
@@ -17,18 +17,16 @@
     "release": "release-me patch"
   },
   "peerDependencies": {
-    "@vitejs/plugin-vue": "^4.0.0",
-    "@vue/compiler-sfc": "^3.2.47",
-    "@vue/server-renderer": "^3.2.47",
-    "vite": "^4.4.9",
-    "vike": "^0.4.147",
-    "vue": "^3.3.4"
+    "@vitejs/plugin-vue": "^4.5.2",
+    "vite": "^4.5.1",
```

Comment:
Should it be something like `^4.5.1 || ^5.0.7` so we support both Vite 4 and 5?

And then we wouldn't need to downgrade Vite in the examples?

Reply:
Done